tests/test_wrapper.sh.in

```diff
@@ -352,6 +352,13 @@ echo "----------------------------------------------------------------------"
 echo "  $sfversion passed tests on OPUS files."
 echo "----------------------------------------------------------------------"
 
+# wavpack-tests
+./tests/wavpack_test@EXEEXT@
+
```

Comment:
Good start on the tests. Please consider adding wavpack tests to other tests that apply as you add features. Eg, add a section to `string_test`.